<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IHM</class>
 <widget class="QMainWindow" name="IHM">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IHM</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : #4a87d9;
font-size : 25px;
color : white;
font-family : monospace;</string>
  </property>
  <widget class="QWidget" name="affichagePrincipal">
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>480</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="menuManuel" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>480</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #b9b9b9;
border-top-right-radius: 50px;
border-bottom-right-radius: 50px;</string>
      </property>
      <widget class="QPushButton" name="boutonDesactive">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>241</width>
         <height>231</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Desktop/Icônes/bouton.png</normaloff>../Desktop/Icônes/bouton.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>225</width>
         <height>225</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="boutonAutoOn">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>320</y>
         <width>141</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Desktop/Icônes/auto2.png</normaloff>../Desktop/Icônes/auto2.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="menuAuto" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>350</width>
         <height>480</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>480</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #a7cdfa;
border-top-right-radius: 50px;
border-bottom-right-radius: 50px;</string>
       </property>
       <widget class="QPushButton" name="boutonActive">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>241</width>
          <height>231</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../Desktop/Icônes/bouton.png</normaloff>../Desktop/Icônes/bouton.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>225</width>
          <height>225</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="boutonAutoOff">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>320</y>
          <width>141</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../Desktop/Icônes/auto.png</normaloff>../Desktop/Icônes/auto.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
       <widget class="QWidget" name="menuVerrouillerPIN" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>371</width>
          <height>511</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color : #4a87d9;</string>
        </property>
        <widget class="QLabel" name="codeCache">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>90</y>
           <width>181</width>
           <height>71</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:60px; 
background-color: transparent;
color: #000000;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="imageDigicode">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>70</y>
           <width>321</width>
           <height>341</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #000000;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../Desktop/digicode.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="boutonValider">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>340</y>
           <width>61</width>
           <height>30</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../Desktop/Icônes/valider.png</normaloff>../Desktop/Icônes/valider.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="bouton9">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>280</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #000000;
background-color: transparent;</string>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bouton6">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>220</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #000000;
background-color: transparent;</string>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bouton3">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>170</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #000000;
background-color: transparent;</string>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bouton8">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>280</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #000000;
background-color: transparent;</string>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bouton0">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>340</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #000000;
background-color: transparent;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bouton5">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>220</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #000000;
background-color: transparent;</string>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bouton2">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>170</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #000000;
background-color: transparent;</string>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bouton4">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>220</y>
           <width>61</width>
           <height>41</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #000000;
background-color: transparent;</string>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bouton1">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>170</y>
           <width>61</width>
           <height>41</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #000000;
background-color: transparent;</string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
        <widget class="QPushButton" name="boutonEffacer">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>330</y>
           <width>81</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../Desktop/Icônes/effacer.png</normaloff>../Desktop/Icônes/effacer.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="bouton7">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>280</y>
           <width>61</width>
           <height>41</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #000000;
background-color: transparent;</string>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
        <zorder>imageDigicode</zorder>
        <zorder>codeCache</zorder>
        <zorder>boutonValider</zorder>
        <zorder>bouton9</zorder>
        <zorder>bouton6</zorder>
        <zorder>bouton3</zorder>
        <zorder>bouton8</zorder>
        <zorder>bouton0</zorder>
        <zorder>bouton5</zorder>
        <zorder>bouton2</zorder>
        <zorder>bouton4</zorder>
        <zorder>bouton1</zorder>
        <zorder>boutonEffacer</zorder>
        <zorder>bouton7</zorder>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="menuPrincipal" native="true">
      <property name="maximumSize">
       <size>
        <width>450</width>
        <height>480</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #4a87d9;</string>
      </property>
      <widget class="QLabel" name="logoTemperature">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>100</y>
         <width>101</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Desktop/temperature.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="logoHumidite">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>180</y>
         <width>101</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Desktop/humidite.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="logoCO2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>250</y>
         <width>111</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Desktop/co2.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="logoPM25">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>310</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Desktop/pm25.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="dateHeure">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>291</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size : 20px;
color : white;
font-family : monospace;</string>
       </property>
       <property name="text">
        <string>16 : 51 | 15 / 02 / 2021</string>
       </property>
      </widget>
      <widget class="QLabel" name="valeurTemperature">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>120</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size : 25px;
</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QLabel" name="valeurHumidite">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>190</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size : 25px;</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QLabel" name="valeurCO2">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>260</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size : 25px;</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QLabel" name="valeurpm25">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>330</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size : 25px;</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QWidget" name="menuVerrouillerHeure" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>451</width>
         <height>491</height>
        </rect>
       </property>
       <widget class="QLabel" name="heure">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>160</y>
          <width>351</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 80px;
</string>
        </property>
        <property name="text">
         <string>16 : 51</string>
        </property>
       </widget>
       <widget class="QLabel" name="date">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>250</y>
          <width>321</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 35px;
</string>
        </property>
        <property name="text">
         <string>15 / 02 / 2021</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="uniteTemp">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>130</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>C°</string>
       </property>
      </widget>
      <widget class="QLabel" name="uniteHumi">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>200</y>
         <width>31</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
      <widget class="QLabel" name="uniteCO2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>270</y>
         <width>68</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>ppm</string>
       </property>
      </widget>
      <widget class="QLabel" name="unitePM25">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>330</y>
         <width>68</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>ppm</string>
       </property>
      </widget>
      <widget class="QPushButton" name="boutonDeco">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>370</y>
         <width>101</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Desktop/deco.png</normaloff>../Desktop/deco.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <zorder>boutonDeco</zorder>
      <zorder>uniteTemp</zorder>
      <zorder>uniteHumi</zorder>
      <zorder>unitePM25</zorder>
      <zorder>uniteCO2</zorder>
      <zorder>logoTemperature</zorder>
      <zorder>logoHumidite</zorder>
      <zorder>logoCO2</zorder>
      <zorder>logoPM25</zorder>
      <zorder>dateHeure</zorder>
      <zorder>valeurTemperature</zorder>
      <zorder>valeurHumidite</zorder>
      <zorder>valeurCO2</zorder>
      <zorder>valeurpm25</zorder>
      <zorder>menuVerrouillerHeure</zorder>
     </widget>
    </item>
   </layout>
   <zorder>menuPrincipal</zorder>
   <zorder>menuManuel</zorder>
  </widget>
  <action name="actionAllumerAuto">
   <property name="text">
    <string>AllumerAuto</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>boutonAutoOff</sender>
   <signal>clicked()</signal>
   <receiver>menuAuto</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boutonAutoOn</sender>
   <signal>clicked()</signal>
   <receiver>menuAuto</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boutonDeco</sender>
   <signal>clicked()</signal>
   <receiver>menuVerrouillerPIN</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>716</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boutonDeco</sender>
   <signal>clicked()</signal>
   <receiver>menuVerrouillerHeure</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>716</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
